varlink/server.py

```diff
@@ -11,28 +8,15 @@
 import string
 import sys
 
-try:
-    from builtins import int
-    from builtins import object
-    from builtins import open
-    from builtins import range
-except ImportError:
-    pass
-
 from .error import (InterfaceNotFound, InvalidParameter, MethodNotImplemented, VarlinkEncoder, VarlinkError,
                     ConnectionError)
```

Comment:
The ConnecitonError implementation in .error and import should go away, this is since 3.3 a subclass of OSError https://docs.python.org/3/library/exceptions.html#ConnectionError

Reply:
Note: also requires cleaning up `__all__` in `varlink/__init__.py`.

The ConnectionError cleanup also applies to BrokenPipError and is why the tests currently fail.